jwst/assign_wcs/util.py

```diff
@@ -278,7 +278,7 @@ def wcs_from_footprints(dmodels, refmodel=None, transform=None, bounding_box=Non
                 fiducial[k] = crval[i]
                 i += 1
 
-    ref_fiducial = compute_fiducial([refmodel.meta.wcs])
+    ref_fiducial = np.array([refmodel.meta.wcsinfo.ra_ref, refmodel.meta.wcsinfo.dec_ref])
```

Comment:
If a reference model is specified on input, then this fiducial should be based at that reference model's reference point, not just a random position in the center of it's bounding box.  It may not make much of a difference, but this will affect the computation of the final output plate scale later and will now accurately describe the plate scale of the desired output reference model.  It may prove worthwhile to add a comment to the code to this effect for later maintenance.